lib/api_browser/app/views/directives/attribute_description/values.html

```diff
@@ -0,0 +1,14 @@
+<dt ng-if="row.value.length == 1">Value</dt>
+<dt ng-if="row.value.length < 5 && row.value.length > 1">Possible values</dt>
+<dt ng-if="row.value.length >= 5">
+  Possible values (<a ng-init="row.isCollapsed = true" ng-click="row.isCollapsed = !row.isCollapsed">{{row.isCollapsed? 'show' : 'hide'}}</a>)
+	</dt>
+</dt>
+
+<dd ng-if="row.value.length == 1"><code>{{row.value[0]}}</code> (this field will always hold this value)</dd>
+<dd ng-if="row.value.length > 1">
+  <div ng-class="{hide: !row.isCollapsed || row.value.length < 5}">{{row.value.slice(0, 4).join(', ')}}, <a ng-init="row.isCollapsed = true" ng-click="row.isCollapsed = !row.isCollapsed">and {{row.value.length - 4}} more</a>.</div>
```

Comment:
`ng-class="{hide: !row.isCollapsed || row.value.length < 5}"` should cover that, no? That means it will be shown if the row is collapsed and the length is greater then 5.